src/checklist/CheckItems/CheckItems.tsx

```diff
@@ -8,7 +8,6 @@ import { ProcosysApiService } from '../../services/procosysApi';
 const CheckItemsWrapper = styled.div`
     display: flex;
     flex-direction: column;
-    margin-bottom: 24px;
     & div:first-of-type {
         margin-top: 0;
     }
```

Comment:
In this file the allItemsCheckedOrNA  in the CheckItemsProps type isn't in the prop list of the CheckItems component. It could probably be removed.